torchci/components/TestInsights.tsx

```diff
@@ -0,0 +1,60 @@
+import { JobData } from "../lib/types";
+import React from "react";
+
+const NOT_SUPPORTED_JOBS = [
+  "android",
+  "bazel",
+  "libtorch",
+  "macos",
+  "rocm",
+];
+
+export default function TestInsightsLink({
+  job,
+  separator,
+}: {
+  job: JobData;
+  separator: string;
+}) {
+  if (job === null) {
+    return (<span></span>);
+  }
+
+  const workflowId = job.htmlUrl?.match(
+    // https://github.com/pytorch/pytorch/actions/runs/3228501114/jobs/5284857665
+    new RegExp("^.+/(.+)/jobs/.+$")
+  )?.[1];
+
+  const jobId = job.logUrl?.match(
+    // https://github.com/pytorch/pytorch/actions/runs/3228501114/jobs/5284857665
```

Comment:
This url doesn't match the regex below?

Reply:
Ah yeah, my copy/paste mistake, the example should be something like `https://ossci-raw-job-status.s3.amazonaws.com/log/9018026324`